.github/workflows/test_chart.yaml

```diff
@@ -0,0 +1,46 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+env:
+  TAG: dev
+  IMAGE: controller
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.1
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
+
+      - name: setupGo
+        uses: actions/setup-go@v4
+        with:
+          go-version: '=1.20.4'
+
+      - name: Run chart-testing (lint)
+        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --validate-maintainers=false
+
+      - name: Build docker image
+        run: MANIFEST_IMG=${{ env.IMAGE }} TAG=${{ env.TAG }} PULL_POLICY=Never make docker-build
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.7.0
+        with:
+          cluster_name: kind
+
+      - name: Add local docker image
+        run: kind load docker-image ${{ env.IMAGE }}:${{ env.TAG }}
+
+      - name: Run chart-testing (install)
+        run: helm install test charts/rancher-turtles/ --post-renderer ./config/kustomize.sh --wait
```

Comment:
what if include manifests that are outside customized like here https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/hack/charts/cluster-api-operator/templates? How the post-renderer script will behave?

Reply:
It will work in tandem

Reply:
so validation won't fail if we add manifests that are not part of kustomize built ones?

Reply:
why not to place `config/kustomize.sh` under `config/chart` which is specific to chart folder?

Reply:
Discussed it with Alex, will work upon existing implementation, where helm templating comes second after kustomize. This implementation does it in the opposite order - first templating helm resources, then applies kustomize from config/default on templated resources. Provides a benefit of doing something like setting default namespace on all resources, including templated ones, but current implementation of release process will have to be updated to reflect that. Otherwise we may end up with two different versions of helm chart - packaged and release, and installed locally from folder.